BookingsApi/BookingsApi.UnitTests/Validation/UpdateHearingParticipantsRequestValidationTests.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Validations;
+using FizzWare.NBuilder;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.Validation
+{
+    public class UpdateHearingParticipantsRequestValidationTests
+    {
+        private UpdateHearingParticipantsRequestValidation _validator;
+        
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _validator = new UpdateHearingParticipantsRequestValidation();
+        }
+
+        [Test]
+        public async Task Should_pass_validation()
```

Comment:
When should it pass validation? Can we test when it won't pass validation?

Reply:
I think the naming convention is fine because it takes multiple things to cause a validation to pass, but only one thing to fail. 

So the tests asserting failure must specify the condition causing the failure, but the test causing it to pass shouldn't otherwise the naming becomes obsolete whenever the validation is changed and you risk ridiculously long names e.g.

Should pass when there is at least one participant and the participant is not a judge and the hearing is not confirmed etc.

For validation logic like this, we don't look at the validation passed test to tell us the criteria, but we look at the failing conditions instead (for reasons above). The passing test is only there as a check to ensure the validation does in fact pass if a developer updates the validation logic.